### DISCUSSION TITLE
Difficulty on Taylor factor

### DISCUSSION THREAD

**Discussion by Ericpan3** [mtex-toolbox/mtex#1658] [Ask Anything] [answered]
Hello, I tried to calculate the Taylor factor following the instruction https://mtex-toolbox.github.io/TaylorModel.html.
While the MATLAB keep showing error: 
![2022-11-18 (2)](https://user-images.githubusercontent.com/80721968/202796289-4a70856b-7c7d-45c9-919a-8698fe7d7658.png)

Could you please let me know where this error from?

**Comment by smallrain1996** [comment, marked as answer]
You don't have a full installation of MATLAB and some toolkits are missing
